frame/democracy/src/lib.rs

```diff
@@ -528,6 +529,53 @@ decl_error! {
 	}
 }
 
+/// Functions for calcuating some weight of dispatchables.
```

Comment:
```suggestion
/// Functions for calcuating the weight of dispatchables.
```
or alternatively "the weight of some dispatchables" if that is what you mean.